CMakeLists.txt

```diff
@@ -24,6 +20,10 @@ include(platform)
 include(GetPrerequisites)
 include(CheckCXXCompilerFlag)
 
+set_policies(
+  TYPE NEW
+  POLICIES CMP0073
+           CMP0074)
```

Comment:
Can we set NEW or OLD individually for each policy ? It would be nice if this generic in that manner, otherwise looks fine.

Reply:
You can add another set_policies call for OLD behaviours. We should only stick with NEW in most cases.